public/locales/en/translations.json

```diff
@@ -18,40 +18,38 @@
     },
     "login": {
       "footer-text": "Don't you have account?",
-      "title": "Login"
-    }
-  },
-  "login-form": {
-    "placeholder": {
-      "email": "Email address",
-      "password": "Password"
-    },
-    "forgot": "Forgot?",
-    "login": "Login"
-  },
-  "password-restore": {
```

Comment:
password-restore or password-reset or reset-password? Let's choose one name for a component and localization.